eduardagoulart/Recommender_system_animes
# Calculate genre similarity

### Problem

Currently, we don't have a way to calculate the similarity between genres

### Solution

Calculate the similarity

### Manual test

- Run `python similarity_anime_dataset/genre_sim.py`
- confirm you have a list with a dictionary as return
